charts/r2devops/templates/configmap.yaml

```
@@ -2,29 +2,43 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  annotations:
+    {{- range $key, $val := .Values.additionalAnnotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
```

Avoid empty `annotation` key if no `additionalAnnotations`:

```suggestion
  {{- with .Values.additionalAnnotations }}
  annotations:
    {{- range $key, $val := . }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
  {{- end }}
```